stores/preferences.ts

```diff
@@ -1,20 +1,36 @@
 import { defineStore } from 'pinia'
 
-type completePurchaseModalState = {
+type CompletePurchaseModalState = {
   isOpen: boolean
   mode: 'shopping-cart' | 'buy-now'
 }
 
+export const DEFAULT_GRID_SECTION = 'explore-gallery'
+export type GridSection = 'profile-gallery' | typeof DEFAULT_GRID_SECTION
```

Comment:
I think an enum would give btter autocomplete
other then that lgtm

Reply:
fixed